src/urllib3/_version.py

```diff
@@ -1,2 +1,2 @@
 # This file is protected via CODEOWNERS
-__version__ = "1.26.5"
+__version__ = "1.26.5+dev"
```

Comment:
Just clarifying that before merging this we're reverting this change?

Reply:
Right, I reverted it.